File: lab-2/src/pokemons/Tirtouga.java
package pokemons;

import moves.*;
import ru.ifmo.se.pokemon.*;

public class Tirtouga extends Pokemon{
    public Tirtouga(String name, int level) {
        super(name, level);
        setStats(54,78,103, 53, 45, 22);
        setType(Type.WATER, Type.ROCK);
        setMove(new Slash(), new RockTomb(), new DoubleTeam());
    }
}

File: lab-2-kotlin/src/pokemons/Tirtouga.kt
package pokemons

import moves.*

import ru.ifmo.se.pokemon.*

open class Tirtouga(name: String?, level: Int) : Pokemon(name, level) {
    init {
        setStats(54.0, 78.0, 103.0, 53.0, 45.0, 22.0)
        setType(Type.WATER, Type.ROCK)
        setMove(Slash(), RockTomb(), DoubleTeam())
    }
}